### PR TITLE
Test/message signing

### LINKED ISSUE #178: Add tests for message signing and passphrase fix
Add E2E tests for above fix

### PR DESCRIPTION
**Pull request by jtsang586**
Closes #178 

Tests to cover recent bug fix for signing message and invalid passphrase

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,7 @@
 ### 🛠 Improvements
 
 - [](https://github.com/vegaprotocol/vegawallet-desktop/pull/) -
+- Test Cover for message signing
```
Comment (mattrussell36) [inline CHANGELOG.md:16]: Can we link the PR and fix the typo 'Test cover' => 'Test coverage'

**Review by mattrussell36 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (dexturr) [inline CHANGELOG.md:16]:
```suggestion
- Test Coverage for message signing
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@
 ### 🛠 Improvements
 
 - [](https://github.com/vegaprotocol/vegawallet-desktop/pull/) -
```
Comment (dexturr) [inline CHANGELOG.md:15]:
```suggestion
- [180](https://github.com/vegaprotocol/vegawallet-desktop/pull/180) - Test Coverage for message signing
```

Comment (jtsang586) [reply CHANGELOG.md]: Thanks Dexter

**Review by dexturr — APPROVED** [review]

**Pull request merged**